uint16 gpioId
---
bool result